DebugSwift/Sources/Helpers/Managers/Theme.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ThemeStyling.swift
+//  DebugSwift
+//
+//  Created by Mochamad Rakha Luthfi Fahsya on 26/01/24.
+//
+
+import Foundation
+
```

Comment:
To run in package, needs:

`import UIKit`

Reply:
i'm forget sorry 😂